src/cfnlint/customRules/Operators.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+def equalsOp(rule, propertyList):
+    """ Process EQUALS operators """
+    if not propertyList:
+        return 'Error - Invalid Resource Type ' + rule.resourceType
```

Comment:
not sure this follows, a resource type given in a rule is not necessarily invalid if the property type is not found in a template being evaluated:

```bash
echo 'AWS::SNS::Topic TopicName EQUALS "B"' > custom_rules.txt

echo \
'Resources:
  Resource:
    Type: AWS::SQS::Queue' > template.yaml
cfn-lint template.yaml

echo \
'Resources:
  Resource:
    Type: AWS::SNS::Topic' > template.yaml
cfn-lint template.yaml
```
```
Invalid Resource Type AWS::SNS::Topic
```

should be considered successful if no properties of that type even need to be evaluated

Reply:
Ah - Fair point. Will remove that check for now. Thinking that could become a configurable error message with one of the future milestones set for this project.

Reply:
Misunderstood original comment. Rephrased message for clarity

Reply:
Commented out, will touch on this again once working on configurable messages